lib/ui/screens/contact_list_screen.dart

```diff
@@ -46,14 +49,15 @@ class _ContactListScreenState extends State<ContactListScreen> {
     }
   }
 
-  navigateDetail(int index) {
-    ///will be modified after merging database to contact_list_screen
-    //  Navigator.push(
-    //     context,
-    //     MaterialPageRoute(
-    //         builder: (context) => ContactDetail(contact: contactList[index])
-    //         )
-    // );
+  _navigateDetail(int index){
+    if (ContactsRepository.getContactInfo(index) != null)
```

Comment:
final Contact? contact = ContactsRepository.getContactInfo(index);
```suggestion
    final Contact? contact = ContactsRepository.getContactInfo(index);
if (contact != null) {
Navigator.push(
          context,
          MaterialPageRoute(
              builder: (context) => 
                  ContactDetailScreen(contact: contact)
              );
}

```